package customexception;

public class CustomExceptionDemo {
    public static void main(String[] args) {
        try {
            System.out.println("");
        }
    }
}
